employee_dialog/lib/Employee profile/employee_preferences.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:employee/Employee profile/employee.dart';
+import 'employee_preferences.dart';
+
+class EmployeePreferences {
```

Comment:
I can see that this is only a dummy class for some dummy data for testing, I believe it's better if you name the class accordingly as well